api/tests/python/tests/todo/349_test_molv2000_charge.py

```diff
@@ -0,0 +1,20 @@
+import os
+import sys
+sys.path.append(os.path.normpath(os.path.join(os.path.abspath(__file__), '..', '..', '..', "common")))
+from env_indigo import *
+
+indigo = Indigo()
+indigo.setOption("ignore-stereochemistry-errors", True)
+
+    
+def testMolv2000Charge(filename):
+    
+    print(relativePath(filename))
+    try:
+        mol = indigo.loadMoleculeFromFile(joinPath(filename))
+        print("loaded.")
```

Comment:
please add charge and isotope for each atom since it was initially required in the story

Reply:
Done. Also output affected fields of Bonds block: order (aka type), and stereo (aka direction).